cupy/manipulation/add_remove.py

```diff
@@ -1,13 +1,21 @@
 import cupy
-
+from cupy.creation.from_data import asanyarray
+from cupy import core
 
 # TODO(okuta): Implement delete
 
 
 # TODO(okuta): Implement insert
 
-
-# TODO(okuta): Implement append
+def append(tup, values, axis=None):
```

Comment:
This function is implemented in https://github.com/cupy/cupy/pull/1802/files#diff-4fd534cefc7099c784eefd47425e197bR13

Please don't add unrelated features in the same PR.

Reply:
Sure, I need it in the insertion part that is why I put it here. I will replace it with the function provided in the link if it is not merged yet